internal/config/git.go

```diff
@@ -91,16 +78,19 @@ func GitClone(dir string, u *url.URL) error {
 	})
 
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to clone repo, error: %v", err)
```

Comment:
can you add the **config:**  prefix to the error messages?

Reply:
is added in the logging (in the place where is called this function